### PR TITLE
[MB-2131] Convert xamarin build process to gradle

### PR DESCRIPTION
**Pull request by madelgi**
Should make it automatically pull down most recent android sdk version for android bindings. Also can maybe move the update version functionality to gradle.

### CODE REVIEW

**Comment by rlepinski** [comment]
👍 

Yes please move the update version stuff into the gradle build system. It would be nice if it could update the other versions based on the airship.properties file.

**Comment by rlepinski** [comment]
👍  Assuming it works.

File: gradlew.bat [hunk 1]
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (BrianBatchelder) [inline gradlew.bat]: Just have to ask - do we ever build on Windows?

Comment (rlepinski) [inline gradlew.bat]: nope, this is just the standard gradle wrapper

Comment (BrianBatchelder) [inline gradlew.bat]: Thought I'd seen it before. Guess it doesn't hurt, but its inclusion might kinda imply we tested it.

Comment (madelgi) [inline gradlew.bat]: i think the idea is to make it so folks on windows can build the repo on their systems as well, even if we'll never do it ourselves

Comment (BrianBatchelder) [inline gradlew.bat]: Ah. My original question should have been "does anyone ever build on Windows?".

Thanks, Max.

**Pull request merged**